sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/objects/objects.scala

```diff
@@ -151,7 +151,7 @@ trait InvokeLike extends Expression with NonSQLExpression {
   final def findMethod(cls: Class[_], functionName: String, argClasses: Seq[Class[_]]): Method = {
     val method = MethodUtils.getMatchingAccessibleMethod(cls, functionName, argClasses: _*)
     if (method == null) {
-      sys.error(s"Couldn't find $functionName on $cls")
+      throw QueryExecutionErrors.methodNotDeclaredError(functionName)
```

Comment:
This file has one more `sys.error`? https://github.com/apache/spark/blob/1bcc51223141236d0f785c65de4c69f13a257033/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/objects/objects.scala#L488

The change looks fine for consistency.

Reply:
Fix together? Or another minor PR as this is a followup only.

Reply:
hm. I think either is okay, but how about assigning a new jira number for replacing all the `sys.error` in this file with `QueryExecutionErrors.xxx` methods?

Reply:
sgtm